Testscripts/Linux/STOR_VHDXResize_GrowFSAfterResize.sh

```diff
@@ -64,11 +64,14 @@ if ! [ "$rerun" = "yes" ]; then
 	if [ $? -ne 0 ]; then
 		LogErr "File-system tools for $fs not present. Skipping filesystem $fs."
 	else
-		#Use -f option for xfs filesystem, but ignore parameter for other filesystems
+		# Use -f/-F option for xfs/ext4 filesystem, but ignore parameter for other filesystems
 		option=""
 		if [ "$fs" = "xfs" ]; then
```

Comment:
if [ "$fs" = "xfs" ] || [ "$fs" = "ext4" ];

Reply:
@xian123 @lizzha  please help me understand these options; -f and -F are different.  I think both cases should be -F. Please let me know.

Reply:
Please man mkfs.xfs and mkfs.ext4, and you'll find that the -F option is not supported by mkfs.xfs.

Reply:
didn't notice they're different options, then I'm OK with the change